static_src/main.js

```diff
@@ -93,7 +92,7 @@ function app(orgGuid, spaceGuid, appGuid) {
   appActions.fetch(appGuid);
   appActions.fetchStats(appGuid);
   ReactDOM.render(
-    <MainContainer initialSpaceGuid={ spaceGuid }>
+    <MainContainer>
       <AppContainer
         initialAppGuid={ appGuid }
```

Comment:
Let's merge now and then open a new PR just for the current app stuff. The reason for this is that there are a lot of upcoming changes in the app container and it will be easier to think about the conflicts with fewer changes.